### ISSUE TITLE
Replace learnr with bookdown?

### ISSUE THREAD

**Issue by soniamitchell** [IBAHCM/RPiR#351]
I wonder if we should just remove the learnr stuff and put everything in bookdown. 
* It would make the code chunks format nicer on the website.
* It would remove dependencies.
* We could instead put the intro stuff (loops and functions) into a stand-alone learnr section if you wanted to.

**Comment by richardreeve** [comment]
I don't think so. I really like the learnr stuff early on, and I think it will expand over time, and this allows flexibility for later stuff. Also, how do you run bookdown inside RStudio? The nice thing about the current setup is that you can access it from the web if you're having computer problems or RStudio if you don't have an internet connection. Also I think the problem with learnr is really just that we're having to use github dev versions, but that will stop being true soon (hopefully!)...

**Comment by soniamitchell** [comment]
If we're going to expand the learnr stuff then that makes sense! 👍
